packages/vite/src/node/cli.ts

```diff
@@ -180,6 +186,7 @@ cli
         base: options.base,
         mode: options.mode,
         configFile: options.config,
+        configLoader: options.configLoader as undefined,
```

Comment:
Why does this have `as undefined`? Is it not needed if `GlobalCLIOptions` has `configLoader?: 'bundle' | 'runner'` instead?